### DISCUSSION TITLE
Face-X community Discussion

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/Face-X#1207] [General]
Feel free to discuss anything!

**Comment by EssamSYD** [comment]
Hi AKSHITA
I'm new in girlscript summer code and contribute with face-x project
I confused about contributing plz I want to know the steps to work in this project on github

**Comment by externalPointerVariable** [comment]
Hi Akshita,
This is Abhishek, I am a contributor in GSSoC'24 and this project was listed there but when I searched through all the issues, I did not see any tags regarding GSSoC not to mention the level of the issues.
Please look into it
